src/file.js

```diff
@@ -24,22 +24,8 @@ Sk.builtin.file = function (name, mode, buffering) {
     } else if (this.name === "/dev/stderr") {
         this.fileno = 2;
     } else {
-        if (Sk.inBrowser) {  // todo:  Maybe provide a replaceable function for non-import files
```

Comment:
OK, Too much multi-tasking for me!

I think I understand now.   Essentially I need to take that code and put it in my own function and configure it as Sk.read  which in my current environment Sk.read would thrown an unimplemented error.

What about the write function?